scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -305,6 +305,39 @@ DFN-8-1EP_2x2mm_P0.5mm_EP0.9x1.5mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-8-1EP_3x3mm_P0.5mm_EP1.65x2.38mm:
+  device_type: 'DFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.analog.com/media/en/technical-documentation/data-sheets/4320fb.pdf#page=10'
+  # ipc_class: 'qfn' # 'qfn_pull_back'
+  body_size_x:
+    nominal: 3
+    tolerance: 0.1
+  body_size_y:
+    nominal: 3
+    tolerance: 0.1
+  lead_width:
+    nominal: 0.25
+    tolerance: 0.05
+  lead_len:
+    nominal: 0.4
+    tolerance: 0.1
+
+  pitch: 0.5
+  num_pins_x: 0
+  num_pins_y: 4
+
+  EP_size_x: 1.65
+  EP_size_y: 2.38
+
+  thermal_vias:
+    count: [2, 3]
+    drill: 0.2
```

Comment:
Is it necessary to enter µVia waters for this relatively large EP?

Reply:
0.2mm is the standard drill size I recall being used throughout. You can see 3 existing definitions in dfn.yaml using it. Does something need to change?

Reply:
Yes, most are 0.2mm.
I tend to ask contributors for 0.3mm (although I forget sometimes) because 0.3mm is often the smallest mechanical hole cheap fabs will drill.

I figure our ThermalVias footprints are mostly used as ad hoc, better-than-nothing thermal management solutions, so 0.3mm seems to be a more reasonable default to me.
As far as I'm aware, most scripted footprints currently use 0.2mm for no other reason than that value being copied and pasted over and over again.

Reply:
Could this be fixed globally? Perhaps if and when this repo is moved to GitLab? It seems rather unrelated to this one PR. Or are you requiring this change on all script updates to be merged starting now?

Reply:
Alright, meeting as is.

Reply:
So, even _knowing_ about the bugs filed against this, and raising it in review, it went in anyway?

Reply:
1. There has been no decision on https://github.com/KiCad/kicad-footprints/issues/2153
2. This FP isn't completely broken but just perhaps less useful than it could be (and a slightly inconvenient FP is definitely better than nothing)
3. As @evanshultz said, fixing this globally when the matter is settled is easy.

So, yes, it went in anyway. We can't keep everything on hold because there might be some marginal issue with it, especially if it's easily fixed afterwards.